roles/openshift_logging_fluentd/templates/2.x/fluentd.j2

```diff
@@ -98,6 +98,8 @@ spec:
           readOnly: true
 {% endif %}
         env:
+        - name: "MERGE_JSON_LOG"
+          value: "{{ openshift_logging_fluentd_merge_json_log }}"
```

Comment:
Is there a default setting for `openshift_logging_fluentd_merge_json_log`?

Reply:
Yeah, seems to be set here to "true" https://github.com/openshift/openshift-ansible/blob/bae78fb8ab91348392c69777a589d3e20c8c5135/roles/openshift_logging_fluentd/defaults/main.yml#L32